cellcom/Cargo.toml

```diff
@@ -8,6 +8,11 @@ license.workspace = true
 repository.workspace = true
 rust-version.workspace = true
 
+# TODO: Switch fully between reqwest for Google and orb-security-utils for Status backends
+[features]
```

Comment:
blocker: we should not use cargo feature flags for customizing backend endpoints. Use `ORB_BACKEND` env var and switch at runtime. or better yet, explicitly accept as an argument the backend type.